packages/documentation/src/stories/misc/migration-guide/migrationv4-5.component.ts

```diff
@@ -24,7 +24,7 @@ export class MigrationV45Component extends LitElement {
         <a
           aria-hidden="true"
           tabindex="-1"
-          href="http://localhost:9000/?path=/docs/c23b1d0b-76b3-4e38-aa76-b10c29bb873f--docs#migration-from-v5-to-v6#migration-from-v4-to-v5"
+          href="#migration-from-v4-to-v5"
```

Comment:
Sadly, this is not working and causes the page to reload with the iframe.html instead of just adding the hash to the current url.
Therefore, it must be reverted to something like `/?path=/docs/c23b1d0b-76b3-4e38-aa76-b10c29bb873f--docs#migration-from-v5-to-v6`.
Please test the links after the update, to ensure they work as expected.
Do this also with all the other migrationv{from}-{to}.component.ts files.